src/main/java/uk/gov/digital/ho/hocs/queue/complaints/CorrespondentType.java

```diff
@@ -3,4 +3,5 @@
 public enum CorrespondentType {
     COMPLAINANT,
     THIRD_PARTY_REP,
+    MIGRATION,
```

Comment:
I'd have thought you'd have wanted your own file for these correspondent types?

Reply:
Removed from here and will create one when we received data finalised migration schema.